src/main/java/com/dsrm/dsrmbackend/validation/annotations/ExistingRoom.java

```diff
@@ -0,0 +1,17 @@
+package com.dsrm.dsrmbackend.validation.annotations;
+
+import com.dsrm.dsrmbackend.validation.validators.ExistingRoomValidator;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.*;
+
+@Target({ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = ExistingRoomValidator.class)
+@Documented
+public @interface ExistingRoom {
+    String message() default "does not exist";
```

Comment:
That is not a good solution. Please remove field name from  com.dsrm.dsrmbackend.controllers.RestResponseEntityExceptionHandler#handleMethodArgumentNotValid and would jest return message name. I would fix other tests messages to follow new pattern.